scss/_reboot.scss

```diff
@@ -727,3 +727,26 @@ progress {
 [hidden] {
   display: none !important;
 }
+
+// Boosted mod
+[class*="-dark"],
+[class*="bg-secondary"] {
+  --link-color: #{$link-color-dark};
+  --link-hover-color: #{$link-hover-color-dark};
+  --caption-color: #{$table-caption-color-dark};
+  --code-color: #{$code-color-dark};
+  --kbd-color: #{$kbd-color-dark};
+  --kbd-bg: #{$kbd-bg-dark};
+  --pre-color: #{$pre-color-dark};
+
+  [class*="bg-"]:not(&):not(.bg-transparent) {
```

Comment:
Not sure about that because in the utilities, we don't have the context of use

Reply:
Rethinking about this, you're right too: you may give context to utilities (only needs the `[class*="-dark"]` parent, after all) but it would lead to duplicate code.